test/docgen/view/__snapshots__/documentation.test.ts.snap

```diff
@@ -13807,6 +13823,14 @@ This is the URL inside the kubeconfig file to use with kubectl
 
 ---
 
+###### Example <a name=\\"Example\\" id=\\"aws-cdk-lib.aws_eks.Cluster.property.clusterEndpoint.example\\"></a>
+
+\`\`\`csharp
+// Example automatically generated from non-compiling source. May contain errors.
+https:
+\`\`\`
```

Comment:
This looks incorrect, but I did some digging and found that the reason this example (and a few others) are being rendered as "code examples" is just due to errors in the older version of `aws-cdk-lib` that's being used as a test fixture, and the examples appear correctly in newer versions of aws-cdk-lib.

It's probably worth updating these test fixtures to newer ones, but I'll leave that open for a later PR.